Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -245,40 +246,55 @@ protected function requireConstraintsImposedByAncestorsAreMet(
         array $parentNodeAggregateIds,
         ContentRepository $contentRepository
     ): void {
-        foreach ($parentNodeAggregateIds as $parentNodeAggregateId) {
-            $parentAggregate = $this->requireProjectedNodeAggregate(
-                $contentStreamId,
-                $parentNodeAggregateId,
-                $contentRepository
-            );
-            try {
-                $parentsNodeType = $this->requireNodeType($parentAggregate->nodeTypeName);
-                $this->requireNodeTypeConstraintsImposedByParentToBeMet($parentsNodeType, $nodeName, $nodeType);
-            } catch (NodeTypeNotFound $e) {
-                // skip constraint check; Once the parent is changed to be of an available type,
-                // the constraint checks are executed again. See handleChangeNodeAggregateType
-            }
```

Comment:
an alternative possibly simpler fix would be to just wrap this check in a condition:

```
if (!$parentAggregate->classification->isTethered()) {
```

the mental overhead having possibly multiple parent nodeaggregates is mindblowing

Reply:
adjusted